src/test/java/core/basesyntax/FruitTransactionTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
+    private static final String APPLE = "apple";
+    private static final String BANANA = "banana";
+    private static final String ORANGE = "orange";
+    private static final String GRAPE = "grape";
```

Comment:
I guess you don`t need same constants in every class

Reply:
But why if they are used in tests? Most likely, then the code would not pass the checkstyle test, if the constants were redundant